internal/policy/policy.go

```diff
@@ -567,7 +572,7 @@ func (s *State) Verify(ctx context.Context) error {
 	}
 
 	if len(delegationEnvelopes) != 0 {
-		return ErrDanglingDelegationMetadata
+		return fmt.Errorf(fmt.Sprint(delegationEnvelopes))
```

Comment:
can you explain this change?

Reply:
Also beware of #219...

Reply:
I suggest leaving `Verify` alone.

Reply:
I was changing that when I was debugging, and I forgot to revert it

Reply:
Fixed